MekHQ/src/mekhq/campaign/market/PersonnelMarket.java

```diff
@@ -157,7 +157,7 @@ public void generatePersonnelForDay(Campaign campaign) {
                     stringBuilder.append("<br>A ");
                 }
                 stringBuilder.append("<b> ")
-                    .append(expLevel)
+                    .append(SkillType.getColoredExperienceLevelName(person.getExperienceLevel(campaign, false)))
```

Comment:
This might be something for another PR, but I think we should overload this method with a version that defaults secondary to false. Given the sheer frequency of `person.getExperienceLevel(campaign, false)` in our codebase, I think that would have value.

Reply:
That sounds like a good idea for another PR, yeah. I'll look at doing that when I clear my current stuff :>.

Reply:
Made a note of it even.